Dockerfile

```diff
@@ -1,8 +1,26 @@
-FROM nginx:alpine
+FROM amd64/nginx:alpine
 MAINTAINER hhendrik@ethz.ch
+MAINTAINER aschnabl@ethz.ch
+
+RUN mkdir /rbb_web
+COPY / /rbb_web
+WORKDIR /rbb_web
+
+RUN apk update
+RUN apk add  --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.7/main/ nodejs=8.9.3-r1
+
+RUN node --version
+RUN npm --version
+RUN npm install yarn 
+RUN ./node_modules/yarn/bin/yarn --version
+
+RUN ./node_modules/yarn/bin/yarn install
+RUN ./node_modules/yarn/bin/yarn build-staging
```

Comment:
Could you make this into a multi-stage build? It's a feature of docker where you can install deps, build a result then copy the result to a new container that won't copy all the layers from building. That way the `node_modules` won't get brought along for the entire container, also you could base the build container off a node container and the deployment one off nginx

https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds

Reply:
Done